spec/integration/v2/jobs_spec.rb

```diff
@@ -38,11 +71,15 @@
   end
 
   context 'GET /jobs/:job_id/log.txt' do
-    it 'returns log for a job' do
-      stub_request(:get, "#{Travis.config.logs_api.url}/logs/#{job.id}?by=job_id&source=api")
-        .to_return(status: 200, body: JSON.dump(content: 'the log'))
+    it 'returns redirects to archived log url' do
```

Comment:
This adds a new spec, in which we ensure that the default request for a `.txt` content (with no `Accept` header) is served a plain text.